src/index.js

```diff
@@ -1,34 +1,28 @@
-import './style.css';
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] While you did a good job implementing the required changes :+1:, it's important to remember not to disable `eslint` inside your document because `eslint` helps to catch errors and improve the quality of your code.